voorhoede/tudelft-spacefinder
# Improve accessible labeling

# Changes

Adds label to the occupancy indicator and translate the labels of the space card navigation buttons.

Also removes some unused code for displaying number of seats in the `SpaceFacilities` component.

# Associated issue

Partly resolves https://trello.com/c/C9CYUm8O/63-accessibility-improvements-for-spacefinder-app-and-map

# How to test

1. Open preview link.
2. The occupancy indicator should now have an accessible name.
3. The copy of the previous and next button in the space card should now be translated.

# Checklist
